app/models/commits_service.rb

```diff
@@ -0,0 +1,8 @@
+class CommitsService
+
+  def self.find_commits(username, repository)
+    response = Faraday.get("https://api.github.com/repos/#{username}/#{repository}/commits?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
@Dpalazzari gotcha! well, i think these are all ways to refactor so keep cruising along 👍